pngencoder.lua

```diff
@@ -1,6 +1,9 @@
 local Png = {}
 Png.__index = Png
 
+PNG_COLOR_TYPE_RGB = 2
```

Comment:
Having these be globals is a bit against the whole isolated module thing. Since I don't think Lua has any nice ways to export named members, it might be best to just get rid of these altogether